servers/republik/modules/crowdfundings/graphql/resolvers/PackageOption.js

```diff
@@ -23,5 +23,8 @@ module.exports = {
         id: membershipId
       })
     }
+  },
+  optionGroup ({ optionGroup, rewardId }) {
```

Comment:
While implementing the frontend I noticed that this causes more problems than it solves. So I'm not actually querying it except for custom pledges (for membership grouping). Please remove this resolver again.